sphinx/builders/latex/__init__.py

```diff
@@ -167,6 +172,31 @@ def init_document_data(self):
                 docname = docname[:-5]
             self.titles.append((docname, entry[2]))
 
+    def init_context(self):
+        # type: () -> None
+        self.context = DEFAULT_SETTINGS.copy()
```

Comment:
The method of preparing this context in the builder is the same as what we do in the `builders/ html.py` code, so I didn't feel uncomfortable with this change